starfish/core/imagestack/parser/crop.py

```diff
@@ -58,6 +58,31 @@ def _add_permitted_axes(self, axis_type: Axes, permitted_axis: int) -> None:
         if axis_type == Axes.ZPLANE and self._permitted_zplanes:
             self._permitted_zplanes.add(permitted_axis)
 
+    def further_crop(self, further_crop_params) -> "CropParameters":
+        if further_crop_params._permitted_chs and \
```

Comment:
You could rewrite the three if-else as one single if-else inside a loop if you want.  It's debatable if it's better.